src/locales/lang/fr.json

```diff
@@ -11,9 +11,9 @@
   "plugins.links.standardHint": "Cliquez pour consulter le site",
   "plugins.search.placeholder": "Taper pour rechercher",
   "plugins.unsplash.photoLink": "Photo",
-  "plugins.weather.high": "High",
-  "plugins.weather.low": "Low",
-  "settings": "Settings",
-  "settings.translationCredits": "Traduction française par Orfaon",
-  "widgets": "Widgets"
+  "plugins.weather.high": "Max",
+  "plugins.weather.low": "Min",
+  "settings": "Préférences",
+  "settings.translationCredits": "Traduction française par Orfaon/ernest33",
+  "widgets": "Gadgets"
```

Comment:
My phone also uses the term "Widgets".

Reply:
> 
> 
> My phone also uses the term "Widgets".

Yes that's true, but this is an "Anglicism" ;-)